parachains/runtimes/assets/statemine/src/weights/xcm/mod.rs

```diff
@@ -234,6 +234,6 @@ impl<Call> XcmWeightInfo<Call> for StatemineXcmWeight<Call> {
 		Weight::MAX
 	}
 	fn unpaid_execution(_: &WeightLimit, _: &Option<MultiLocation>) -> Weight {
-		XcmGeneric::<Runtime>::unpaid_execution()
+		Weight::from_ref_time(3_111_000 as u64)
```

Comment:
Why is this done here?

Reply:
obsolete, like we discussed, regenerating new pallet_xcm_benchmarks::generic with real benchmark for unpaid_execution